internal/serverinstall/k8s.go

```diff
@@ -364,6 +364,45 @@ func (i *K8sInstaller) Install(
 	}
 	s.Done()
 
+	s = sg.Add("Waiting for the bootstrap process to finish...")
+	var bootJobName string
+	err = wait.PollImmediate(2*time.Second, 5*time.Minute, func() (bool, error) {
+		// the label we use for LabelSelector is set here
+		// https://github.com/hashicorp/waypoint-helm/blob/d2f6de6e9010b94da84f37eeaca4a8190a439060/templates/bootstrap-job.yaml#L8
+		jobs, err := clientset.BatchV1().Jobs(i.config.namespace).List(ctx, metav1.ListOptions{
+			LabelSelector: "app.kubernetes.io/instance=waypoint",
+		})
+		if err != nil {
+			return false, nil
+		}
+		// the job we are searching for is prefixed with `waypoint-bootstrap`
```

Comment:
```suggestion
		// NOTE(krantzinator): the job we are searching for is prefixed with `waypoint-bootstrap`
```

This might make it easier to show up when people are looking for notes or heads up comments